test/TestCaseReader.cpp

```diff
@@ -39,9 +39,9 @@ TestCaseReader::TestCaseReader(string const& _filename):
 
 string const& TestCaseReader::source()
 {
-	if (m_sources.size() != 1)
+	if (m_sources.sourceMap.size() != 1)
 		BOOST_THROW_EXCEPTION(runtime_error("Expected single source definition, but got multiple sources."));
-	return m_sources.begin()->second;
+	return m_sources.sourceMap.begin()->second;
```

Comment:
This is equivalent, would make it clearer that you're returning the main file and also works as a sanity check that this file is present in the source map:
```suggestion
	return m_sources.sourceMap.at(m_sources.mainSourceFile);
```